### PR TITLE
Styleguide

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: kitsune/sumo/static/sumo/scss/config/_variables.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+$styleguide_path: "../../../../../../styleguide-examples/";
```
Comment (akatsoulas) [inline kitsune/sumo/static/sumo/scss/config/_variables.scss]: Can't we sue the full path here?

File: kss-config.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "source": "kitsune/sumo/static/sumo/scss",
+  "destination": "static/build/styleguide",
+  "//": "The css and js paths are URLs relative to the style guide destination.",
+  "css": [
+    "static/css/screen.css"
+  ],
+  "//": [
+    "Change the key to js and remove this note when JS is compiled with the frontend. old: js/bundle.js"
+  ],
+  "homepage": "../../../../../styleguide-examples/styleguide-index.md",
```
Comment (akatsoulas) [inline kss-config.json]: Maybe a full path?

File: kss-config.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (akatsoulas) [inline kss-config.json]: Do we need to have this top level?

Comment (akatsoulas) [reply kss-config.json]: I would move it inside the folder with all the kss files.

File: kitsune/sumo/static/sumo/scss/config/_variables.scss (hunk 1 above)
Comment (johngian) [reply kitsune/sumo/static/sumo/scss/config/_variables.scss]: full path from where ?

File: kss-config.json (hunk 2 above)
Comment (johngian) [reply kss-config.json]: same

File: kss-config.json (hunk 3 above)
Comment (johngian) [reply kss-config.json]: yes because of `package.json`

**Pull request merged**